app/models/concerns/annotating.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Annotating
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def annotated
```

Comment:
It might be interesting to receive a list of supported annotations, e.g. `annotated :managed, :whatever` and add validations and maybe dynamic accessors based on those names.

But well, let's not overcomplicate :sweat_smile:

Reply:
It's not a bad idea. The main reason I have to not do it is laziness. But I'll do it if you think it's needed. We can do it in another PR as well.

Reply:
Yeah, that's a valid reason. Also, no need to complicate the things unnecessarily. If this was some kind of a public API, we'd think more carefully. As it's just for internal use, I think we can just leave it as it is.

Reply:
yeah, no need to overengineer it now we don't know it will be needed. Shouldn't be hard to change if ever needed.